LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/no-painting-while-render-blocked-expected.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<title>View transitions: Rendering suppression prevents painting (ref)</title>
+<link rel="help" href="https://github.com/WICG/view-transitions">
```

Comment:
The help link can be dropped from the ref. (Side note: you might want to have the `-ref.html` file as well in this patch to export easily)